src/main/webapp/dev/DatePickerJsTest.js

```diff
@@ -0,0 +1,22 @@
+QUnit.module('datepicker.js');
+
+QUnit.test('getMinDateForPublishDate(visibleDate)', function(assert) {
+    assert.deepEqual(getMinDateForPublishDate(new Date('10/13/2014')), new Date('10/13/2014'), 'returns visibleDate');
```

Comment:
The thing is that I want to show the function do nothing but just return the parameter, you can rephrase it by yourself.